CONTRIBUTING.md

```diff
@@ -115,3 +115,10 @@ This DCO replaces the previously used CLA ("Contributor License Agreement") as w
     - Note that the UI5 developers have many duties. So, depending on the required effort for reviewing, testing, and clarification, this may take a while.
 1. Once the change has been approved and merged, we will inform you in a comment.
 1. Celebrate! 🎉
+
+### Multi Branch Development
+There are phases when UI5 Tooling has already started developing a new major versions of it's projects but does not has released the new major version. This sitatuation requires a special handling of pull requests / commits if the actual coding change should be part of the current and the new major version.
+
```

Comment:
I would really like a short example here. Something like "Current version : v3; New Major: v4".

Otherwise looks great!

Reply:
done. If you're fine with the current state. I will assign UA.